addon/components/hyper-table-v2/index.hbs

```diff
@@ -1,4 +1,12 @@
 <div class="hypertable-container" ...attributes>
+  <div class='hypertable__upper-header'>
+    <div class="left-side">
+      Search
```

Comment:
Internationalise ?

Reply:
I don't understand :/

Reply:
@Miexil it's going to change.i think Julien did it just to separate the sections, but it's going to be a named-block anyway